inc/Utf8/Sort.php

```diff
@@ -0,0 +1,154 @@
+<?php
```

Comment:
There are other changed files outside the `inc` folder:

    strcmp()
    --------
    lib/plugins/acl/admin.php:307 --> intl_strcmp()

    sort()
    ------
    bin/wantedpages.php:86           --> intl_sort()
    lib/plugins/acl/admin.php:592    --> intl_sort()
    lib/plugins/acl/admin.php:611    --> intl_sort()
    lib/plugins/authpdo/auth.php:602 --> intl_sort()

    ksort()
    -------
    bin/wantedpages.php:80             --> intl_ksort()
    lib/plugins/acl/admin.php:612      --> intl_ksort()
    lib/plugins/acl/admin.php          --> [:613 new] intl_ksort()
    lib/plugins/authad/auth.php:735    --> intl_ksort()
    lib/plugins/authpdo/auth.php:635   --> intl_ksort()
    lib/plugins/authplain/auth.php:301 --> intl_ksort()
    lib/plugins/authplain/auth.php     --> [:338 new] intl_ksort()

    asort()
    -------
    lib/plugins/authad/adLDAP/classes/adLDAPContacts.php:274 --> intl_asort()
    lib/plugins/authad/adLDAP/classes/adLDAPGroups.php:527   --> intl_asort()
    lib/plugins/authad/adLDAP/classes/adLDAPUsers.php:571    --> intl_asort()
    lib/plugins/authad/adLDAP/classes/adLDAPUsers.php:634    --> intl_asort()
    lib/plugins/authldap/auth.php:416                        --> intl_asort()